src/pages/club-events/index.jsx

```diff
@@ -1,10 +1,132 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { H4, H6, P } from "../../common/components/elements/Text";
+import Modal from "../../common/components/Modal";
+import { ViewCards } from "../../common/components/ViewCards";
 import PageLayout from "../../common/layout/PageLayout";
+import { AxiosGet } from "../../lib/axios";
+
+const ModalDetails = ({
+  EventOrganizer,
+  Deadline,
+  EventDesc,
+  EventDay,
+  EventTime,
+  RegisLink,
+  tags,
+}) => (
+  <div className="flex flex-col items-center gap-5">
+    {EventOrganizer && (
+      <div className="w-full text-center text-black pt-0.5 flex justify-around items-center gap-1">
+        <div className="w-1/2">
+          <P className="font-semibold bg-primary-200 p-2 m-2 rounded-lg">
+            Organizer: {EventOrganizer}
+          </P>
+          <P className="font-semibold bg-primary-200 p-2 m-2 rounded-lg">
+            Event Day: {EventDay.substring(0, 10)}
+          </P>
+        </div>
+        <div className="w-1/2">
+          <P className="font-semibold bg-primary-200 p-2 m-2 rounded-lg">
+            Deadline: {Deadline.substring(0, 10)}
+          </P>
+          <P className="font-semibold bg-primary-200 p-2 m-2 rounded-lg">
+            Event Time: {EventTime}
+          </P>
+        </div>
+      </div>
+    )}
```

Comment:
why checking for event organizer only. it should be for all right.
for ex: check the same for event-day, deadline, event-time also separately.